src/validate.ts

```diff
@@ -0,0 +1,79 @@
+'use strict';
+
+chai.use(require('chai-json-schema-ajv'));
+
+function generateRandom(min, max){
+    //helper function for obtaining a random requestId
+    return Math.floor(Math.random() * (max-min) + min).toString();
+}
+
+
+function generateSyncReq(){
```

Comment:
Nit: If these functions are part of the API you expose (not sure currently), "Request" should be spelled out.